lib/csv_export.rb

```diff
@@ -70,4 +70,20 @@ def download_equipment_data
     download_zip([categories, models, items],
                  "EquipmentData_#{Time.zone.now.to_s(:number)}")
   end
+
+  # NOTE: this method depends on ActionController
+  def download_category_data(cat)
+    categories = [[cat], %w(name max_per_user max_checkout_length
```

Comment:
Should we hardcode these string arrays, either into this library or maybe into the model as a class method? We're calling them several times in the various methods here.